services/autorust/codegen/src/codegen_operations.rs

```diff
@@ -83,33 +83,56 @@ pub fn create_client(modules: &[String], endpoint: Option<&str>) -> Result<Token
             credential: std::sync::Arc<dyn azure_core::auth::TokenCredential>,
             endpoint: Option<String>,
             scopes: Option<Vec<String>>,
+            options: azure_core::ClientOptions,
         }
 
         #default_endpoint_code
 
         impl ClientBuilder {
+            #[doc = "Create a new instance of `ClientBuilder`."]
+            #[must_use]
             pub fn new(credential: std::sync::Arc<dyn azure_core::auth::TokenCredential>) -> Self {
                 Self {
                     credential,
                     endpoint: None,
                     scopes: None,
+                    options: azure_core::ClientOptions::default(),
                 }
             }
 
+            #[doc = "Set the endpoint."]
+            #[must_use]
             pub fn endpoint(mut self, endpoint: impl Into<String>) -> Self {
                 self.endpoint = Some(endpoint.into());
                 self
             }
-
+            #[doc = "Set the scopes."]
```

Comment:
```suggestion

            #[doc = "Set the scopes."]
```

Reply:
Also, it took me a second to figure out what "scopes" refers to. Can we make the docs clearer here?